app/presenters/publishing_api_presenters/detailed_guide.rb

```diff
@@ -83,4 +84,25 @@ def unpublishing_explanation
       Whitehall::GovspeakRenderer.new.govspeak_to_html(item.unpublishing.explanation)
     end
   end
+
+  def applicable_nations
+    all_nations = ['england', 'scotland', 'wales', 'northern_ireland']
+    inapplicabilities = item.nation_inapplicabilities
+    applies_to_all_nations = inapplicabilities.empty?
+
+    if applies_to_all_nations
+      nations = []
+    else
+      excluded_nations = inapplicabilities
+        .map(&:nation)
+        .map(&:name)
+        .map { |n| n.downcase.tr(' ', '_') }
```

Comment:
As of https://github.com/alphagov/govuk-content-schemas/pull/300/commits/55c6fef7af0850a5e7bdd4e8b0adaefee4e6ee18, I don't believe we should be doing this anymore, and instead just sending down the pretty printed nations.